### PR TITLE
Update authentication.md

### PR DESCRIPTION
**Pull request by cvweiss**
Added the missing CREST scope endpoints listed in https://community.eveonline.com/news/patch-notes/patch-notes-for-eve-online-citadel/

### CODE REVIEW

File: docs/crest/authentication.md [hunk 1]
```diff
@@ -11,24 +11,73 @@ When designing your application you will need to decide what information you req
...
+Allows an application to read your character's notifications.
+
+### characterResearchRead
+Allows an application to read your character's research information.
+
+### characterSkillsRead
+Allows an application to read your character's skills.
+
+### characterWalletRead
+Allows an applicatin to read your character's wallet (ambiguous on transactions or journal)
```
Comment (CarbonAlabel) [inline docs/crest/authentication.md]: Nothing ambiguous here, it lets you read both transactions and journal as well.

Comment (cvweiss) [inline docs/crest/authentication.md]: Thanks, I didn't look so I wasn't sure.

**Comment by Blacksmoke16** [comment]
While you are updating this can also add 

remoteClientUI

Allows an application to control the UI of your EVE Online client.

**Comment by cvweiss** [comment]
I removed the comment about ambiguity between wallet and journal. Should be ready now.

**Comment by cvweiss** [comment]
I'll put that one in my next PR

**Comment by CarbonAlabel** [comment]
You're also missing the corp scopes.

**Comment by cvweiss** [comment]
You're right, corp scopes are missing since I was adding character only. Was going to get to the corp scopes later on a page meant specifically for corp scopes. The scope of this PR is for characters in the link I initially provided.

Scrap this PR or accept it please.

**Comment by polarina** [comment]
Thanks! Cherry-picked as ddec95dc1959f696e5703ac8ed3e1953689dfefb.

**Pull request closed without merging**